auction-app-frontend/src/constants.js

```diff
@@ -168,3 +173,24 @@ export const EMPTY_TABS = [
 
 export const HIGHEST_BID_MESSAGE = "Congratulations! You are the highest bidder.";
 export const LOWER_BID_MESSAGE = "There are higher bids than yours. You could give a second try.";
+
+export const ADD_ITEM_FORMS = [
+  { id: "details", label: "Details" },
+  { id: "prices", label: "Prices" },
+  { id: "shipping", label: "Shipping" },
+];
+
+export const ADD_ITEM_FORMS_MAP = {
+  DETAILS: "details",
+  PRICES: "prices",
+  SHIPPING: "shipping",
+};
+
+export const successfulRegistrationMessage = "Registration successful, please login.";
```

Comment:
Every single object here is in format NAME_NAME_NAME
Kepp it for variable the same
`SUCCESSFUL_REGISTRATION_MESSAGE`